CHANGELOG.md

```diff
@@ -2,13 +2,17 @@
 
 All notable changes to `WP Curate` will be documented in this file.
 
+## 2.2.3 - 2024-08-22
```

Comment:
I'd argue this is more than a patch and could/should be `2.3.0`

Reply:
Yeah, I suppose that is accurate. I've made that update. Thanks for flagging